.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 repos:
--   repo: https://github.com/ambv/black
+-   repo: https://github.com/psf/black
```

Comment:
I included a related minor change: 
psf/black is the new location of black and now https://github.com/ambv/black redirects there.